src/less/chart-legend/chart-legend.less

```diff
@@ -0,0 +1,123 @@
+.chart-legend {
+    display: inline-block;
+}
+.chart-legend--hidden {
+    border: 0;
+    clip: rect(0, 0, 0, 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+}
+.chart-legend__table {
+    flex-shrink: 0;
+}
+.chart-legend ul {
+    list-style: none;
+    margin: 0;
+    padding-inline-start: 24px;
+}
+.chart-legend__item {
+    line-height: 2rem;
+    position: relative;
+    transition: opacity 0.5s;
+    &--patternfill::before {
```

Comment:
Even though `less` offers this internal cascading feature, we typically opt to write out the entire selector. This would apply to all of the internal ones as well.

Reply:
Agree with @ArtBlue here. Having the entire selector makes it possible to find with text search. This should be added to our style-guide: https://github.com/eBay/skin/blob/master/CONTRIBUTING.md#style-guide